docs/graphql/manual/api-reference/graphql-api/query.rst

```diff
@@ -160,7 +160,7 @@ Simple object
      - Value
      - ``path`` argument of ``json``/``jsonb`` follows simple `JSONPath specification <https://github.com/json-path/JsonPath>`_. However, prefix symbol ``$.`` is optional.
 
-E.g.
+*Example*
```

Comment:
Was using italics intentional instead of bold. Just generally asking, it's fine either way.

Reply:
It was intentional because I needed the bolt for this nesting level: https://deploy-preview-4844--hasura-docs.netlify.app/graphql/manual/api-reference/graphql-api/query.html#tableorderby. And by making the examples italic, it's consistent again :D